dependencies.props

```diff
@@ -76,6 +79,14 @@
       <BuildInfoPath>$(BaseDotNetBuildInfo)projectn-tfs/$(DependencyBranch)</BuildInfoPath>
       <CurrentRef>$(ProjectNTfsCurrentRef)</CurrentRef>
     </RemoteDependencyBuildInfo>
+    <RemoteDependencyBuildInfo Condition="'$(DisableProjectNTfsAutoUpgrade)' != 'true'" Include="ProjectNTfs-TestILC">
```

Comment:
If I remember correctly, that condition is there so that there's a very obvious switch to temporarily freeze the ProjectN version or manually change it.